openreview/conference/builder.py

```diff
@@ -200,11 +277,14 @@ def set_reviewers(self, emails):
         return self.webfield_builder.set_reviewer_page(self.id, group)
 
     def set_authors(self):
-        notes_iterator = tools.iterget_notes(self.client, invitation = self.get_submission_id())
+        notes_iterator = self.get_submissions(details = 'original')
 
         for n in notes_iterator:
             group = self.__create_group('{conference_id}/Paper{number}'.format(conference_id = self.id, number = n.number), self.id)
-            self.__create_group('{number_group}/{author_name}'.format(number_group = group.id, author_name = self.authors_name), self.id, n.content.get('authorids'))
+            authorids = n.content.get('authorids')
+            if n.details and n.details['original']:
+                authorids = n.details['original']['content']['authorids']
+            self.__create_group('{number_group}/{author_name}'.format(number_group = group.id, author_name = self.authors_name), self.id, authorids)
```

Comment:
This section could use a comment.  I'm not sure what's going on - how is authorids used?

Reply:
It is creating the author groups with the real author emails as members. The authorids could come from a single blind submission or double blind, that is what the if is doing.